Octokit/Models/Response/RepositoryStar.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Represents additional information about a star (such as creation time)
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class RepositoryStar
+    {
+        public RepositoryStar() { }
```

Comment:
The empty `ctors`. I ask because we've gone back and forth on how the response models should be structured in the past - previously the properties were `public set`-ers, but then we switched over to `protected set` to indicate which were required.

cc @haacked for feels